framework/db/Migration.php

```diff
@@ -552,9 +538,36 @@ public function dropCommentFromColumn($table, $column)
      */
     public function dropCommentFromTable($table)
     {
-        echo "    > drop comment from table $table ...";
-        $time = microtime(true);
+        $time = $this->beginCommand("drop comment from table $table");
         $this->db->createCommand()->dropCommentFromTable($table)->execute();
-        echo ' done (time: ' . sprintf('%.3f', microtime(true) - $time) . "s)\n";
+        $this->endCommand($time);
+    }
+
+    /**
+     * Prepares for a command to be executed, and outputs to the console
+     * 
+     * @param string $description the description for the command, to be output to the console.
+     * @return float the time before the command is executed, for the time elapsed to be calculated.
+     * @since 2.0.13
+     */
+    protected function beginCommand($description)
```

Comment:
I would rather add a single method `output()` or `log()` instead of 2 separated.
Calculating of the execution time should remain outside it.